pkg/prog/prog.go

```diff
@@ -55,7 +54,6 @@ func newFlagSet(stderr io.Writer, f *Flags) *flag.FlagSet {
 	fs.Usage = func() { usage(stderr, fs) }
 
 	fs.StringVar(&f.Log, "log", "", "a file to write debug log to")
```

Comment:
Update the description for `-log` that it is ignored when `-daemon` is specified, since the daemon always writes logs to stdout.

Reply:
Done. However, I think this is a silly nitpick since manually launching the daemon is dangerous, not officially supported, and hopefully irrelevant in the future when the daemon is eliminated.

Reply:
> I think this is a silly nitpick since manually launching the daemon is dangerous, not officially supported

Your premise is true from a user's perspective, but not true from a developer's perspective. You may not like the daemon, but as long as it exists as a developer I prefer making it easy to debug it and launching it manually is a valuable technique to do that.